ui/packages/catalog-ui-search/src/main/webapp/react-component/utils/fetch/fetch.tsx

```diff
@@ -39,8 +42,16 @@ if (process.env.NODE_ENV !== 'production') {
   }
 }
 
+const cacheBust = (urlString: string) => {
+  const { query, ...rest } = url.parse(urlString)
+  return url.format({
+    ...rest,
+    search: '?' + qs.stringify({ ...qs.parse(query), _: Date.now() }),
+  })
+}
+
 export default function(url: string, { headers, ...opts }: Options = {}) {
-  return fetch(url, {
+  return fetch(cacheBust(url), {
     credentials: 'same-origin',
     cache: 'no-cache',
```

Comment:
I suppose we could update this to `no-store` now since our `fetch` will never use the cache.